nrf_fuel_gauge/CHANGELOG.rst

```diff
@@ -9,6 +9,20 @@ Changelog
 
 All notable changes to this project are documented on this page.
 
+Main Branch
+***********
+
+Changes
+=======
+
+* Update to library version 0.9.1.
+* The :c:func:`nrf_fuel_gauge_ttf_get` function needs to know when in constant current charger mode.
```

Comment:
Does the function now get the information? Maybe needs rephrasing?

Reply:
Rephrased to `The :c:func:nrf_fuel_gauge_ttf_get/ function now uses charger state as argument instead of maximum charge current.`